tools/lsc/src/main.cpp

```diff
@@ -178,18 +204,41 @@ int main(int argc, const char **argv)
 
     LSLuaState::initCommandLine(argc, argv);
 
-#ifdef LOOM_ENABLE_JIT
-    printf("LSC - JIT Compiler\n");
-#else
-    printf("LSC - Interpreted Compiler\n");
-#endif
-
     bool runtests      = false;
     bool runbenchmarks = false;
     bool symbols       = false;
 
     const char *rootBuildFile = NULL;
     const char *sdkRoot = NULL;
+
+    for (int i = 1; i < argc; i++)
+    {
+        if (!strcmp(argv[i], "--log-type"))
+        {
+            i++;
+            if (i >= argc)
+            {
+                LSError("--log-type option requires the log type (default|cli|runtime) to be specified next");
```

Comment:
It toggles between output as a cli tool (just normal output) and output as a runtime asset agent logger (prefixed with `  compiler  ` so it fits along with other messages).